ethcore/src/snapshot/account.rs

```diff
@@ -246,6 +268,7 @@ mod tests {
 			balance: 123456789.into(),
 			storage_root: KECCAK_NULL_RLP,
 			code_hash: KECCAK_EMPTY,
+			code_version: 0.into(),
```

Comment:
Can you please add one more test for encoding accounts with not zero code-version?